roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -1,34 +1,34 @@
 ---
 - name: make sure an installation origin was chosen
   fail:
-    msg: "choose an installation origin"
+    msg: "choose an installation origin between repository, distro and local\nValid ceph_origin are: {{ valid_ceph_origins }}"
   when:
-    - ceph_origin != 'upstream'
-    - ceph_origin != 'distro'
-    - ceph_origin != 'local'
+    - ceph_origin not in valid_ceph_origins
   tags:
     - package-install
 
-- name: make sure an installation source was chosen
+- name: make sure a repository was chosen
   fail:
-    msg: "choose an upstream installation source or read https://github.com/ceph/ceph-ansible/wiki"
+    msg: "choose a repository installation source: {{ valid_ceph_repository }} or read https://github.com/ceph/ceph-ansible/wiki"
```

Comment:
can we link to docs.ceph.com/ceph-ansible instead of the wiki now?

Reply:
definitely!